cli/src/commands/init.rs

```diff
@@ -198,33 +198,33 @@ impl Opts {
 	}
 
 	async fn update_gitignore(&self, term: &Term) -> Result<()> {
-		if !git::check_ignore(Path::new(".rivet/")).await? {
+		if !git::check_ignore(Path::new(".env")).await? {
 			if self.recommend
 				|| self.update_gitignore
-				|| term::Prompt::new("Add .rivet/ to .gitignore?")
-					.docs(".rivet/ holds secrets and local configuration files that should not be version controlled")
-					.docs_url("https://rivet.gg/docs/general/concepts/dot-rivet-directory")
+				|| term::Prompt::new("Add .env to .gitignore?")
+					.docs(".env holds the develpoment token that should not be version controlled")
 					.default_value("yes")
-					.bool(term).await?
+					.bool(term)
+					.await?
 			{
 				let mut file = fs::OpenOptions::new()
 					.write(true)
 					.append(true)
 					.open(".gitignore")
 					.await?;
-				file.write_all(b"\n### Rivet ###\n.rivet/\n.env\n").await?;
+				file.write_all(b"\n.env\n").await?;
 
 				ensure!(
-					git::check_ignore(Path::new(".rivet/")).await?,
-					"updated gitignore does not ignore Rivet files"
+					git::check_ignore(Path::new(".env")).await?,
+					"updated gitignore does not ignore .env"
 				);
```

Comment:
Should this also be a check before writing to the .gitignore to make sure it's not getting added multiple times?

Reply:
That's what `git::check_ignore(Path::new(".env"))` is intended to do. This way, it catches wildcards and we don't have to do any weird parsing on the .gitignore regex.

Reply:
Oops it totally is, didn't look up that far :)